tests/flow_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -29,3 +29,69 @@ const beep = (data /*: Object*/) => {};
 const beep = (data /*: Object*/, secondData /*: Object*/) => {};
 "
 `;
+
+exports[`object_type_annotation.js 1`] = `
+"type Props = // (DispatchProps & StateProps); WHY DON'T YOU WORK FLOW!!!!!!!!!
+{
+  isPlaying: boolean,
+};
+
+type Props = { // (DispatchProps & StateProps); WHY DON'T YOU WORK FLOW!!!!!!!!!
+  isPlaying: boolean
+};
+
+type Props = {
+  // (DispatchProps & StateProps); WHY DON'T YOU WORK FLOW!!!!!!!!!
+  isPlaying: boolean
+};
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+type Props =
+  // (DispatchProps & StateProps); WHY DON'T YOU WORK FLOW!!!!!!!!!
```

Comment:
Ok, I was originally matching it how the `ObjectExpression` version of this behaves, but I guess:

```js
// (DispatchProps & StateProps); WHY DON'T YOU WORK FLOW!!!!!!!!!
type Props = {
  isPlaying: boolean
};
```

Is desired output?